### DISCUSSION TITLE
NEXTCLOUD_MEMORY_LIMIT are available only to root, but not www-data

### DISCUSSION THREAD

**Discussion by brccabral** [nextcloud/all-in-one#4264] [Questions] [answered]
### Steps to reproduce
1. execute `docker run` with `NEXTCLOUD_MEMORY_LIMIT=2048M`
2. the memory is assigned to the `root` user in the container, not to `www-data`
3. run `sudo -u www-data php -i | grep memory` and get `memory_limit => 128M`
4. run as root `php -i | grep memory` and get `memory_limit => 2048M`

### Expected behavior 
User `www-data` should have `2048M` as passed in.

### Actual behavior 
Only root has the memory increase, but apps that need more memory are executed as `www-data`, thus it should get the change as well

### Host OS 
Ubuntu 23.10

#### Nextcloud AIO version 
Nextcloud AIO v7.12.1

#### Current channel 
"You are running the latest channel"

#### Other valuable info 
![image](https://github.com/nextcloud/all-in-one/assets/130455/c9124ff7-1abf-4f77-a30d-68f323c0c520)

**Comment by szaimen** [comment, marked as answer]
Hi, the correct command is `sudo docker exec --user www-data -it nextcloud-aio-nextcloud bash -c "php -i | grep memory"`
